api/v1alpha1/ztunnel_types.go

```diff
@@ -28,10 +28,10 @@ const (
 type ZTunnelSpec struct {
 	// +sail:version
 	// Defines the version of Istio to install.
-	// Must be one of: v1.24.0 or latest.
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,order=1,displayName="Istio Version",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:fieldGroup:General", "urn:alm:descriptor:com.tectonic.ui:select:v1.24.0", "urn:alm:descriptor:com.tectonic.ui:select:latest"}
-	// +kubebuilder:validation:Enum=v1.24.0;latest
-	// +kubebuilder:default=v1.24.0
+	// Must be one of: v1.24.1, v1.24.0, v1.23.3, v1.23.2, v1.22.6, v1.22.5, v1.21.6, latest.
```

Comment:
We had uid issue running ztunnel on OpenShift prior 1.24 . My upstream fix was merged in a 1.24.0 release.
So I don't think we could include 1.23 or earlier version.